packages/dev-server/dev-config.ts

```diff
@@ -66,8 +66,8 @@ export const devConfig: VendureConfig = {
             assetUploadDir: path.join(__dirname, 'assets'),
         }),
         DefaultSearchPlugin.init({ bufferUpdates: true, indexStockStatus: false }),
-        BullMQJobQueuePlugin.init({}),
-        // DefaultJobQueuePlugin.init(),
```

Comment:
With this line, I am not able to start the dev-server locally without errors. (Docs don't mention anything about a local redis instance)

Reply:
Yeah that's fine. I just use the BullMQ job queue when I develop because if I ever need to debug DB queries, it is was easier without the polling noise caused by the DefaultJobQueuePlugin.